react/components/Modal/index.js

```diff
@@ -143,6 +145,8 @@ Modal.propTypes = {
   children: PropTypes.node.isRequired,
   /** Center the modal (for small content) */
   centered: PropTypes.bool,
+  /** Container in which the modal is rendered */
+  container: PropTypes.object,
```

Comment:
What happens if you don't this prop?

Reply:
It defaults to a <div> appended to `document.body` by the library behind
https://github.com/pradel/react-responsive-modal/blob/master/src/modal.js#L69